acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/pubsub/NewPendingTransactionAcceptanceTest.java

```diff
@@ -169,6 +169,7 @@ public void subscriptionToMinerNodeMustReceivePublishEvent() {
   }
 
   @Test
+  @Ignore
```

Comment:
I think these tests are showing a genuine failure.  The bug says "too far behind" not "behind in any respect."  These messages are getting dropped because the full node in a process runner is about 3-4 blocks behind.  We don't see this in the thread runner because there is no process and JVM startup overhead.